### DISCUSSION TITLE
Error while trying to save page - Errorcode 413

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6090] [Error / Bug Report]
- ubuntu 20.04
- docker image

While editing a page and clicking on save the following error message is displayed:
![image](https://user-images.githubusercontent.com/107103943/214572700-20f743d0-a539-48da-b417-f32ae73192cf.png)

I figured that the problem should be something about the allowed file size of a page, because It only occures when I add 2 or more screenshots to the page. 1 Screenshot seems to still be within the file size limit. The problem only occurs on this particular article that I am writing (the article is the longest i have written so far and with most screenshots). So what I tried is trippled the allowed bytes for a page on the Security Section of the Admin Area. I also increased the upload file count from 20 to 50 (although i didnt think that it had to do something with that):
![image](https://user-images.githubusercontent.com/107103943/214576782-0d50fbea-5329-46e7-8600-709ef7112f73.png)

The problem still remains. Any help?

**Comment by fnafox** [comment]
You need to adjust your reverse proxy configuration, apache/nginx isn't allowing you to upload files above a certain size
In Apache, edit `httpd.conf` by adding the line `LimitRequestBody 1073741824` (in bytes) or whichever size you want, then reload the daemon.
For nginx you need to edit `nginx.conf` by adjusting the variable `client_max_body_size 2M;` to whatever size you want and reload the daemon.

**Reply by ghost** [reply]
I am not using any apache/nginx reverse proxy server, cause I have installed Wiki.js with the docker image as per instructions: https://docs.requarks.io/en/install/docker
On the ubuntu virtual machine there is only the 3 docker containers (wiki.js; wiki.js upgrade companion and the wiki.js database)

**Reply by NGPixel** [reply]
Try to restart the wiki container (`docker restart wiki`).

**Reply by ghost** [reply]
Restarted it. Problem still remains.

Im on Version 2.5.294 btw.

**Reply by ghost** [reply]
@NGPixel is there any additional advice that you can give?
